src/main/java/sirius/biz/web/BizController.java

```diff
@@ -705,7 +705,7 @@ protected void redirectToSaneReferer(final WebContext webContext, Pattern patter
      * @see #redirectToSaneReferer(WebContext, Pattern, String)
      */
     protected static Pattern createPatternForOverviewAndDetailRoutes(String overviewRoute, String detailRoutePrefix) {
-        return Pattern.compile("^(" + Pattern.quote(detailRoutePrefix) + "\\w+|" + Pattern.quote(overviewRoute) + ")$");
+        return Pattern.compile("^(" + Pattern.quote(detailRoutePrefix) + "[0-9a-zA-Z_\\-]+|" + Pattern.quote(overviewRoute) + ")$");
```

Comment:
`/user-accounts` is a biz route using a dash 😇

Reply:
I am not talking about generic routes, but routes using identifiers or counters. These are AFAIK usually restricted to even `A-Z0-9`.